crates/sdk/src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub mod env;
 pub mod event;
 mod returns;
 pub mod state;
-mod sys;
+pub mod sys;
```

Comment:
please revert this change, providing a direct, unsafe interface to `sys` is not part of our objectives.

we export safe interfaces via `env`

Reply:
Thanks for the info - this has actually gone now due to the removal of the `write_to_register()` function.